readthedocs/rtd_tests/tests/test_domains.py

```diff
@@ -99,6 +97,28 @@ def test_public_domain_not_allowed(self):
             f'{settings.PUBLIC_DOMAIN} is not a valid domain.'
         )
 
+    def test_domain_with_path(self):
+        form = DomainForm(
+            {'domain': 'domain.com/foo/bar'},
+            project=self.project,
+        )
+        self.assertTrue(form.is_valid())
+        domain = form.save()
+        self.assertEqual(domain.domain, 'domain.com')
+
+    def test_invalid_domains(self):
+        invalid = [
+            'python..org',
+            # '****.foo.com', current validator says this is valid :shrug:
```

Comment:
```suggestion
            # FIXME: '****.foo.com', current validator says this is valid :shrug:
```

Can we add a test for domains with path also, like `domain.com/path/to/something`?

Reply:
The above test is testing that

Reply:
:upside_down_face: